django/db/backends/postgresql/base.py

```diff
@@ -186,16 +186,16 @@ def get_connection_params(self):
                     self.ops.max_name_length(),
                 )
             )
-        conn_params = {}
+        conn_params = {"client_encoding": "UTF8"}
```

Comment:
This value is never used because `conn_params` is unconditionally overwritten below.